ReactNativeClient/lib/MdToHtml.js

```diff
@@ -21,6 +21,7 @@ const setupLinkify = require('./MdToHtml/setupLinkify');
 const hljs = require('highlight.js');
 const markdownItAnchor = require('markdown-it-anchor');
 const markdownItToc = require('markdown-it-toc-done-right');
+const markdownItMultiMd = require('markdown-it-multimd-table');
```

Comment:
This should still be in the plugins section. any reason why you pulled it out of there?

Reply:
Never mind. My bad

Reply:
The plugins section is only for plugins that don't have options. But since there are 2 plugins that require this special treatment, I'll update so everything can go in plugins.

Reply:
No, it's fine. I made a mistake. Same was done for toc.

Reply:
I missed you comment, I've already changed it. I think this is a positive change since things will be more standard for future development.

Reply:
It definitely is. I just didn't want to create more work for you.